lib/commands/new.js

```diff
@@ -19,14 +19,15 @@ module.exports = Command.extend({
   works: 'outsideProject',
 
   availableOptions: [
-    { name: 'dry-run',    type: Boolean, default: false, aliases: ['d'] },
-    { name: 'verbose',    type: Boolean, default: false, aliases: ['v'] },
-    { name: 'blueprint',  type: String,  default: 'app', aliases: ['b'] },
-    { name: 'skip-npm',   type: Boolean, default: false, aliases: ['sn'] },
-    { name: 'skip-bower', type: Boolean, default: false, aliases: ['sb'] },
-    { name: 'skip-git',   type: Boolean, default: false, aliases: ['sg'] },
-    { name: 'yarn',       type: Boolean },
-    { name: 'directory',  type: String,                  aliases: ['dir'] },
+    { name: 'dry-run',     type: Boolean, default: false, aliases: ['d'] },
+    { name: 'verbose',     type: Boolean, default: false, aliases: ['v'] },
+    { name: 'blueprint',   type: String,  default: 'app', aliases: ['b'] },
+    { name: 'skip-npm',    type: Boolean, default: false, aliases: ['sn'] },
+    { name: 'skip-bower',  type: Boolean, default: false, aliases: ['sb'] },
+    { name: 'skip-git',    type: Boolean, default: false, aliases: ['sg'] },
+    { name: 'welcome',     type: Boolean, default: true },
```

Comment:
since `welcome` by itself isn't that descriptive it would probably make sense to add a `description` here too